docs/installation.md

```diff
@@ -87,7 +87,7 @@ Sublime text users can install both [LSP](https://packagecontrol.io/packages/LSP
 ### Emacs
 *⚠ basedpyright has not been tested on this editor. if you run into issues with these instructions, please raise an issue.*
 
-Emacs users can install [eglot](https://github.com/joaotavora/eglot) or [lsp-mode](https://github.com/emacs-lsp/lsp-mode) with [lsp-pyright](https://github.com/emacs-lsp/lsp-pyright).
+Emacs users can install [eglot](https://github.com/joaotavora/eglot), [lsp-bridge](https://github.com/manateelazycat/lsp-bridge) or [lsp-mode](https://github.com/emacs-lsp/lsp-mode) with [lsp-pyright](https://github.com/emacs-lsp/lsp-pyright).
```

Comment:
should we just remove the other links and also remove that "basedpyright has not been tested on this editor" warning above?

Reply:
Yes, basedpyright works fine, should remove warning above.